src/Catty/Helpers/Util/Util.h

```diff
@@ -110,103 +110,103 @@ if (__functor) __functor(__VA_ARGS__);  \
 
 + (CGFloat)statusBarHeight;
 
-+ (CATransition*)getPushCATransition;
++ (CATransition* _Nonnull)getPushCATransition;
 
-+ (ProjectLoadingInfo*)lastUsedProjectLoadingInfo;
++ (ProjectLoadingInfo* _Nonnull)lastUsedProjectLoadingInfo;
```

Comment:
This method is forwarding to `projectLoadingInfoForProjectDirectoryName` which can be null, thus this method must also be `_Nullable`.

Reply:
As written on slack the linter does not accept _Nullable here due to the usage of lastProjectLoadingInfo.

Reply:
This reflects a potential cause for crash. However, since this involves quite a few other changes, we will have to fix this problem in a separate ticket.